data_safe_haven/resources/workspace/ansible/desired_state.yaml

```diff
@@ -170,6 +170,12 @@
         dest: /etc/xdg/xfce4/terminal/
         mode: '0444'
 
+    - name: Disable light locker (screen lock)
+      ansible.builtin.copy:
+        content: "Hidden=true"
+        dest: /etc/xdg/autostart/light-locker.desktop
+        mode: '0644'
+
```

Comment:
Is it definitely the case that we want to clobber this file? I'm 90% sure it doesn't exist by default but have you checked?

Reply:
It doesn't exist by default :+1:.